src/FetchRandomUser.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import styles from "./User.module.css";
+import Usercomp from "./Usercomp.js";
+
+export default class FetchRandomUser extends Component {
+  state = {
+    loading: true,
+    people: null
+  };
+
+  async componentDidMount() {
+    const url = "https://api.github.com/users";
+    const response = await fetch(url);
```

Comment:
make use of axios instead of fetch try to understand difference between fetch and axios